tests/eeid_plugin/host/host.c

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <stdio.h>
+
+#include <openenclave/attestation/plugin.h>
+#include <openenclave/attestation/sgx/eeid_verifier.h>
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+
+#if defined(_WIN32)
+#include <ShlObj.h>
+#include <Windows.h>
+#endif
+
+#include "../../../host/sgx/quote.h"
+#include "../test_helpers.h"
+#include "eeid_plugin_u.h"
+
+#define SKIP_RETURN_CODE 2
+
+void host_verify(
+    uint8_t* evidence,
+    size_t evidence_size,
+    uint8_t* endorsements,
+    size_t endorsements_size)
+{
+    printf("====== running host_verify.\n");
+
+    oe_claim_t* claims = NULL;
+    size_t claims_size = 0;
+    OE_TEST(
+        oe_verify_evidence(
+            evidence, evidence_size, NULL, 0, NULL, 0, &claims, &claims_size) ==
+        OE_OK);
+
+    oe_free_claims_list(claims, claims_size);
+
+    claims = NULL;
+    claims_size = 0;
+
+    // Test with endorsements currently fails.
+    OE_UNUSED(endorsements);
+    OE_UNUSED(endorsements_size);
+    // OE_TEST(
+    //     oe_verify_evidence(
+    //         evidence,
+    //         evidence_size,
+    //         endorsements,
+    //         endorsements_size,
+    //         NULL,
+    //         0,
+    //         &claims,
+    //         &claims_size) == OE_OK);
```

Comment:
Why is this commented out?

Reply:
See comment. It fails and I'm hunting the bug, but I haven't identified what exactly the problem is yet.